### DISCUSSION TITLE
add release notes to git remote tag from command line

### DISCUSSION THREAD

**Discussion by robindutta** [community/community#22260] [New to GitHub]
<p>We use tags on our private git repo projects to denote “stable” points in the project’s development.  </p>
<p>Our final stage after merging the code in our local master is to create a tag and push it to the remote.</p>
<p>For example, if we have a 1.0 release I use command: </p>
<p>git tag -a 1.0 -m “1.0”</p>
<p>Apparently the <strong>release notes</strong> are a separate field (metadata?) than the tag message, so I have to go to the Tags tab, click the pencil icon to edit and write my release notes there for others to see what is in the release.  </p>
<p>Is there no way to add the release notes from the command line?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Since the “Releases” feature is part of GitHub and not git itself, no, there isn’t a way to set the release notes using the git command-line tool. You can, however, use the <a href="https://hub.github.com" rel="nofollow noopener"><code>hub</code> command-line tool</a> to create a release with release notes from the command line.</p>
<p>I hope that helps!</p>
